src/components/Routes/index.tsx

```diff
@@ -25,6 +26,7 @@ const Routes: React.FC = () => {
         <Route path="/login" element={<LoginScreen />} />
         <Route path="/oauth/naver" element={<LoginScreen />} />
         <Route path="/email/verify" element={<EmailVerifyScreen />} />
+        <Route path="/changepassword" element={<ChangePasswordScreen />} />
```

Comment:
아 이거 api 쪽에서 메일을 `/passwords/reset?token=...` 형식으로 링크를 보내고 있어서 변경이 필요해보입니다!
```suggestion
        <Route path="/passwords/reset" element={<ChangePasswordScreen />} />
```